### ISSUE TITLE
exported>imported composite entities all names changed to ShortGUID

### ISSUE THREAD

**Issue by ghramsey** [MattFiler/OpenCAGE#253]
Not sure this is a bug or by design.    I seem to remember something happened a few months back that broke names too.
I finally got to try the composite export and import function. Seems to work OK so far as import (I've yet to test the function of the imported composite) but all the imported entities have their names stripped.  And the new UI has not yet re-enabled editing names(?).

**Comment by MattFiler** [comment]
Entity renaming has been re-enabled on staging now (#246) - but will look at porting entity names with composites too 👍

**Comment by ghramsey** [comment]
Something I just noticed.  It seems all user-added entities have had their named removed and replaced with their shortguid.
![image](https://github.com/MattFiler/OpenCAGE/assets/51859157/e5376c66-f482-46f7-9695-c7e786e3870b)

The four Xenomorph entities were named Alien_1, Alien_2, Alien_3, Alien_4 and those names were also removed.

**Comment by MattFiler** [comment]
Yeah vanilla ones shouldn't lose their name, it'll only be custom ones

**Comment by ghramsey** [comment]
I scrambled a bit to figured what was what is my custom naming of things like save stations and a few button entities.  
Nothing major. Thank's to Prokill's nifty coordinate tool though I could track the item to it's coordinates and rename it right.

I need to ask him to expand that to be a Cheat Engine replacement and also add the coordinate offsets for the 
story script and other composites.

**Comment by MattFiler** [comment]
Gonna include this in #247
